java/ql/src/Security/CWE/CWE-730/PolynomialReDoS.qhelp

```diff
@@ -103,6 +101,33 @@
 
 	</example>
 
+    <example>
+        <p>
+            Sometimes it is unclear how a regular expression can be rewritten to
+            avoid the problem. In such cases, it often suffices to limit the
+            length of the input string. For instance, the following complicated 
+            regular expression is used to match numbers, and on some non-number 
+            inputs it can have quadratic time complexity:
+        </p>
+
+        <sample language="java">
+Pattern.matches("^(\\+|-)?(\\d+|(\\d*\\.\\d*))?(E|e)?([-+])?(\\d+)?$", str); </sample>
+
+        <p>
+            It is not immediately obvious how to rewrite this regular expression 
+            to avoid the problem. However, it might be fine to limit the length
```

Comment:
I think having an authoritative tone would help increase trust in the docs here, WDYT about replacing 'it might be fine', e.g. with something like this?

```suggestion
            to avoid the problem. However, you can mitigate performance issues by limiting the length
```